sr_description/hand/xacro/finger/biotac/biotac.urdf.xacro

```diff
@@ -154,20 +154,20 @@ xmlns:geom="http://playerstage.sourceforge.net/gazebo/xmlschema/#geom">
         <inertia ixx="0.0" ixy="0.0" ixz="0.0" iyy="0.0" iyz="0.0"
         izz="0.0" />
       </inertial>
-      <visual>
+      <!--visual>
```

Comment:
thanks for clarifying. However, commenting the visual is strange and should be checked upstream in moveit collision checking. 
In your chain redesign, why don't you have rh_ffmiddle --(J1 blocked at 20)---> rh_ffdistal --(fixed)---> rh_fftip ? 
I cannot recall why UHAM had done that dummy link thing. I know the J1 must be there to generate some joint_states output for J1 too, but I don't remember the point of having the dummy link. Now your chain is strange, distal is on the side. tip should be the center of the distal tip, but because you want to have J1 in the chain, you need now to have tip after dummy.